### DISCUSSION TITLE
Is it feasible to support BPMN 2.0?

### DISCUSSION THREAD

**Discussion by bibo4** [danielgerlag/workflow-core#906] [Ideas]
It seems BPMN 2.0 is a popular and standardized DSL which is adopted by many software projects:
https://en.wikipedia.org/wiki/List_of_BPMN_2.0_engines
https://www.omg.org/spec/BPMN/2.0/About-BPMN/

The danielgerlag/workflow-core project seems to be the most popular open source C# workflow engine, but it requires developers and users to invest in its specific JSON and YAML field formats

Is it be feasible to let this project support BPMN 2.0? 
Perhaps a library to convert to and from JSON\YAML file formats and BPMN 2.0s XML format?

The biggest benefits will be the ability to import and export workflows between other system. For example https://github.com/bpmn-io/bpmn-js is the most popular open source tool for drawing BPMN diagrams and there have been numerous request on this project for some UI tools to visualize workflows

**Comment by hudingwen** [comment]
Has it come true?
